subcommand/webhook-cert-manager/command.go

```diff
@@ -180,89 +179,78 @@ func (c *Command) Run(args []string) int {
 // MutatingWebhooksConfigs and Secrets when a new Bundle is available on the channel.
 func (c *Command) certWatcher(ctx context.Context, ch <-chan cert.MetaBundle, clientset kubernetes.Interface, log hclog.Logger) {
 	var bundle cert.MetaBundle
-	retryDuration := defaultRetryDuration
 	for {
 		select {
 		case bundle = <-ch:
 			log.Info(fmt.Sprintf("Updated certificate bundle received for %s; Updating webhook certs.", bundle.WebhookConfigName))
 			// Bundle is updated, set it up
 
-		case <-time.After(retryDuration):
+		case <-time.After(defaultRetryDuration):
 			// This forces the mutating ctrlWebhook config to remain updated
-			// fairly quickly. This is done every 30 minutes to ensure the certificates
-			// are in sync. Because the certificate and key are being read from a secret,
-			// this does not have to be processed as aggressively as the 1 sec time in
-			// the connect inject cert watcher.
+			// fairly quickly. helm upgrades will rewrite the contents of the
+			// CA bundle which will not be in sync with the certs in the system.
+			// This fast reconcile ensure the system recovers fairly quickly in case
+			// the secret or MWC gets deleted or reset.
 
 		case <-ctx.Done():
 			// Quit
 			return
 		}
 
-		if err := c.reconcileCertificates(ctx, clientset, bundle, log); err != nil {
```

Comment:
I think I like still keeping this code in `reconcileCertificates` so we don't have those `continue`'s. They seem error prone to me